packages/tilda/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/lanoxx/tilda"
+TERMUX_PKG_DESCRIPTION="A Gtk based drop down terminal for Linux and Unix"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@ELWAER-M"
+TERMUX_PKG_VERSION="1.5.0"
+TERMUX_PKG_SRCURL="https://github.com/lanoxx/tilda/archive/refs/tags/tilda-${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="f664c17daca2a2900f49de9eb65746ced03c867b02144149ef21260cbcd61039"
+TERMUX_PKG_DEPENDS="glib, gtk3, libvte, libconfuse, libx11, gettext, automake, autoconf"
```

Comment:
Remove gettext, automake and autoconf. They are not needed as runtime dependencies and not used during build time. Packages are cross-compiled and build tools installed in Termux prefix are not used and can't be anyway.

Reply:
Well, gettext maybe still necessary.